modules/nodes-pods-configmap-creating-from-files.adoc

```diff
@@ -6,18 +6,58 @@
 [id="nodes-pods-configmap-creating-from-files_{context}"]
 = Creating a config map from a file
 
-You can create a config map from a file.
+You can create a config map from a file by using the `--from-file` flag. You can pass the `--from-file` option multiple times to the CLI.
 
-.Procedure
+You can also specify the key to set in a config map for content imported from a file by passing a `key=value` expression to the `--from-file` option. For example:
 
-The following example procedure outlines how to create a config map from a file.
+[source,terminal]
+----
+$ oc create configmap game-config-3 --from-file=game-special-key=example-files/game.properties
+----
 
 [NOTE]
 ====
 If you create a config map from a file, you can include files containing non-UTF8 data that are placed in this field without corrupting the non-UTF8 data. {product-title} detects binary files and transparently encodes the file as `MIME`. On the server, the `MIME` payload is decoded and stored without corrupting the data.
 ====
 
-You can pass the `--from-file` option multiple times to the CLI. The following example yields equivalent results to the creating from directories example.
+.Prerequisite
+
+* You must have a directory with some files that contain the data with which you want to populate a config map.
++
+The following procedure uses these example files: `game.properties` and `ui.properties`:
++
+[source,terminal]
+----
+$ cat example-files/game.properties
+----
++
+.Example output
+[source,terminal]
+----
+enemies=aliens
+lives=3
+enemies.cheat=true
+enemies.cheat.level=noGoodRotten
+secret.code.passphrase=UUDDLRLRBABAS
+secret.code.allowed=true
+secret.code.lives=30
+----
++
+[source,terminal]
+----
+$ cat example-files/ui.properties
+----
++
+.Example output
+[source,terminal]
+----
+color.good=purple
+color.bad=yellow
+allow.textmode=true
+how.nice.to.look=fairlyNice
+----
+
+.Procedure
 
 . Create a config map by specifying a specific file:
```

Comment:
I would make this procedure a bullet since there is only one step. I see you have multiple steps in the verification so I wonder if it would be confusing to have 2 step 1's here.